### DISCUSSION TITLE
Any chance of somebody standing up a Discord Server?

### DISCUSSION THREAD

**Discussion by jeeftor** [blakeblackshear/frigate#7383] [Ideas]
It would be nice to have a discord available - although I guess most folks are using the Home Assistant discord cameras topic maybe?

**Comment by NickM-27** [comment]
This has been discussed previously https://github.com/blakeblackshear/frigate/issues/153

The problems with discord is that it's not conducive to support because the conversations overlap each other, people often don't provide config and other things which the GitHub issue templates prompt, and are also much less searchable than GitHub.

**Reply by jeeftor** [reply]
Thanks ... I just thought it would be nice :)
